packages/fx-core/src/plugins/resource/aad/plugin.ts

```diff
@@ -424,18 +425,24 @@ export class AadAppForTeamsImpl {
     TelemetryUtils.init(ctx);
     Utils.addLogAndTelemetry(ctx.logProvider, Messages.StartGenerateArmTemplates);
 
-    const result: ArmTemplateResult = {
-      Parameters: JSON.parse(
-        await fs.readFile(
-          path.join(
-            getTemplatesFolder(),
-            TemplatePathInfo.BicepTemplateRelativeDir,
-            Bicep.ParameterFileName
-          ),
-          ConstantString.UTF8Encoding
-        )
-      ),
-    };
+    const solutionSettings = ctx.projectSettings?.solutionSettings as AzureSolutionSettings;
+    const capabilities = solutionSettings.capabilities;
+    let result: ArmTemplateResult | undefined = undefined;
+
+    if (capabilities.includes(TabOptionItem.id) || capabilities.includes(BotOptionItem.id)) {
```

Comment:
Except bot and tab, do we need to consider other azure resource type such as sql, function?

Reply:
Currently there won't be any azure resource except aad that exists without Tab or Bot capability.
We plan to use the hasAzureResource function, but this function includes AAD. I will confirm with Huajie with this, and will not include this change in this pr.

Reply:
Sure, I created another pr to update hasAzureResource function~